ecoscope/plotting/plot.py

```diff
@@ -268,3 +268,41 @@ def plot_seasonal_dist(ndvi_vals, cuts, bandwidth=0.05, output_file=None):
     if output_file:
         fig.write_image(output_file)
     return fig
+
+
+def stacked_bar_chart(data, layout_kwargs=None):
+    """
+    Creates a stacked bar chart from the provided EcoPlotData object
+    Parameters
+    ----------
+    data: ecoscope.Plotting.EcoPlotData
+        The data to plot, counts categorical data.y_col values for data.x_col
+    layout_kwargs: dict
+        Additional kwargs passed to plotly.go.Figure(layout)
+    Returns
+    -------
+    fig : plotly.graph_objects.Figure
+        The plotly bar chart
+    """
+    # TODO cleanup EPD defaults
+    data.style.pop("mode")
+
+    fig = go.Figure(layout=layout_kwargs)
+
+    x_axis_name = data.x_col
+    y_axis_name = data.y_col
+
+    agg = data.grouped[y_axis_name].count().to_frame("count").unstack(fill_value=0).stack().reset_index()
+
+    x = agg[x_axis_name].unique()
+    for category in agg[y_axis_name].unique():
+        fig.add_trace(
```

Comment:
Sorry for my ignorance. Why can't we use `bar` from `plotly.express` directly?

Reply:
Specifically with a count of values it doesn't seem like `px` does what we need.
This way also allows finer grained style control which is ideal.